applications/ecallmgr/src/ecallmgr.hrl

```diff
@@ -178,6 +178,11 @@
                                ,{<<"Caller-ID-Number">>, <<"effective_caller_id_number">>}
                                ,{<<"Callee-ID-Name">>, <<"effective_callee_id_name">>}
                                ,{<<"Callee-ID-Number">>, <<"effective_callee_id_number">>}
+                               ,{<<"effective_callee_id_number">>, <<"effective_callee_id_number">>}
```

Comment:
@lazedo these should not be required, if they are we are leaking!  I suspect this was due to the CCV issue patched earlier.